layouts/partials/feedback.html

```diff
@@ -2,8 +2,8 @@
 <div id="pre-footer">
   <h2>{{ T "feedback_heading" }}</h2>
     <p class="feedback--prompt">{{ T "feedback_question" }} </p>
-    <button class="btn btn-primary mb-4 feedback--yes">{{ T "feedback_yes" }}</button>
-    <button class="btn btn-primary mb-4 feedback--no">{{ T "feedback_no" }}</button>
+    <button class="btn mb-4 feedback--yes">{{ T "feedback_yes" }}</button>
```

Comment:
Not sure if this is the preferred way to handle customizing the buttons.
Could override in _variables_project.scss instead of adding to _custom.scss.

Reply:
I'd prefer if we used Bootstrap's classes rather than customizing CSS personally! ✨

Reply:
Yeah, I agree. Though, the current styling is not great.

Reply:
I left the feedback buttons as oval. Could set the `border-radius`.
For now, leaving as is.